app_spec/zomes/blog/code/src/blog.rs

```diff
@@ -55,13 +40,10 @@ pub fn handle_hash_post(content: String) -> JsonString {
         )
     );
 
-    match hdk::entry_address(&post_entry) {
-        Ok(address) => AddressResponse{address}.into(),
-        Err(hdk_error) => hdk_error.into(),
-    }
+    hdk::entry_address(&post_entry)
 }
 
-pub fn handle_create_post(content: String, in_reply_to: HashString) -> JsonString {
+pub fn handle_create_post(content: String, in_reply_to: Option<Address>) -> ZomeApiResult<Address> {
```

Comment:
Good, I was hoping someone would wrap this in `Option`

Reply:
reminds me that this is a breaking change, `""` will error now. it must be `null` or some value.

Reply:
oh, except that it isn't really breaking because it's just an example haha